From a4ec5e39c29d7a0c57bb46b8978d0b0f9b23ebf4 Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Sat, 13 Sep 2014 15:06:39 +0200
Subject: [PATCH 17/64] prefs: enable "Send a Do-Not-Track header"

Set enable_do_not_track to true by default.

The web community has pointed out (in e.g.
https://news.ycombinator.com/item?id=9483958 ) that defaulting to
DNT=on is "a questionable plan". I agree with that feeling
that DNT becomes useless if everybody has it on.

However, Iridium is order of magnitudes away from "everybody"
currently. We shall reevaluate this in time again.

[The same ycombinator post also quotes section 6.3 of the
https://tools.ietf.org/html/draft-mayer-do-not-track-00#section-6
draft: "It MUST NOT transmit OPT-IN without explicit user consent."
This is ok, because the UA is not sending OPT-IN; it is sending
OPT-OUT.]
---
 chrome/browser/ui/browser_ui_prefs.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/browser/ui/browser_ui_prefs.cc b/chrome/browser/ui/browser_ui_prefs.cc
index 978db3500449..3ed02ec2394d 100644
--- a/chrome/browser/ui/browser_ui_prefs.cc
+++ b/chrome/browser/ui/browser_ui_prefs.cc
@@ -73,7 +73,7 @@ void RegisterBrowserUserPrefs(user_prefs::PrefRegistrySyncable* registry) {
   registry->RegisterDictionaryPref(prefs::kAppWindowPlacement);
   registry->RegisterBooleanPref(
       prefs::kEnableDoNotTrack,
-      false,
+      true,
       user_prefs::PrefRegistrySyncable::SYNCABLE_PREF);
 #if !defined(OS_CHROMEOS) && !defined(OS_ANDROID)
   registry->RegisterBooleanPref(prefs::kPrintPreviewUseSystemDefaultPrinter,
-- 
2.15.0

